content/en/docs/faq/acme.md

```diff
@@ -126,7 +109,41 @@ Events:
 Progress about the state of each challenge will be recorded either as Events
 or on the Challenge's `status` block (as shown above).
 
+In case of DNS01 you will find any errors from your DNS provider here.
+
+Both HTTP01 and DNS01 go through a "self-check" first before cert-manager presents the challenge to the ACME provider.
+This is done not to overload the ACME provider with failed challenges due to DNS or loadbalancer propagations.
+The status of this can be found in the Status block of the describe:
+```console
+$ kubectl describe challenge 
+[...]
+Status:
+  Presented:   true
+  Processing:  true
+  Reason:      Waiting for http-01 challenge propagation: failed to perform self check GET request 'http://example.com/.well-known/acme-challenge/_fgdLz0i3TFiZW4LBjuhjgd5nTOkaMBhxYmTY': Get "http://example.com/.well-known/acme-challenge/_fgdLz0i3TFiZW4LBjuhjgd5nTOkaMBhxYmTY: remote error: tls: handshake failure
+  State:       pending
+[...]
+```
+
+In this example our HTTP01 check fails due a network issue.
+You will also see any errors coming from your DNS provider here.
+
+### HTTP01 troubleshooting
+First of all check if you can see the challenge URL from the public internet, if this does not work check your Ingress and firewall configuration as well as the service and pod cert-manager created to solve the ACME challenge.
+If this does work check if your cluster can see it too. It is important to test this from inside a Pod. If you get a connection error it is suggested to check the cluster's network configuration.
+
+#### Got 404 status code
+If your challenge self-check fails with a 404 not found error. Make sure to check the following:
+* you can access the URL from the public internet
+* the the ACME solver pod is up and running
+* use `kubectl describe ingress` to check the status of the HTTP01 solver ingress. (unless you use `acme.cert-manager.io/http01-edit-in-place`, then check the same ingress as your domain)
```

Comment:
I think because you've not added white space before the first list item, and no whitespace after the *last* list item, this renders weirdly: https://deploy-preview-222--cert-manager.netlify.app/docs/faq/acme/#got-404-status-code

Reply:
Just checked and it's fixed now!